src/lib/modernExtend.ts

```diff
@@ -116,9 +121,14 @@ export function setupConfigureForReporting(
 }
 
 export function setupConfigureForBinding(cluster: string | number, clusterType: 'input' | 'output', endpointNames?: string[]) {
-    const configure: Configure = async (device, coordinatorEndpoint) => {
+    const configure: Configure = async (device, coordinatorEndpoint, definition) => {
         if (endpointNames) {
-            const endpointsMap = new Map<string, boolean>(endpointNames.map((e) => [e, true]));
+            const definitionEndpoints = definition.endpoint(device);
+            const endpointIds: string[] = [];
```

Comment:
`const endpointIds = endpointNames.map((e) => definitionsEndpoints[e])`

Would this work?